tutorials/units-and-integration/units-and-integration.ipynb

```diff
@@ -47,9 +62,9 @@
QtX8IlTNL7aelxahct4czC8uRGlxIeYXFV5/XlyA0uJCFM8p\nQHFhAEUFAS4nRjkvm8krCCBynKjXHHda10tlaTU0eg1/9IPXMT4xhYfvXZGJU5ALigoDuH1xBW5f\nfOMY5uSUontoFOf6ruJcv/XoHhzD5ZFx9I6M4WzfFbx6th+9I+Mzklw8hQFBUWEARYUBK6GZpFZU\nWIDCgCAQEBSINX4XEOtREOX49WOCgAABEUAAgZUcw39byQ3Pr5dd/9vLdix8JFr96bLryTeyXq7J\nxb9P68pLsO3XGjJ6jmwmryoHx53U9UyZiLTCmtiB5cuXx3lpbBOTildCvfiPm9fgpoW8vyvXFQQE\niyrmYlHFXDTFqaeqGBmfxMjYBIbHJjA8OjH9fGR8AsNjkxi7NonxySmMXZvC+OQUxidsD/P12MQU\nJqemMKnWjNbJKcWkKiampjA2oTccn9Lr5aqYPhZ5j7aqQqefm3+htuf2Mo2oF+c9bMeQo7fU5ehl\nYc3C0pxKXrFaK71J1PVS2TRVbQPQBgBNTU1J/VxWzi/C849+EGUlc5J5OeUoEUGp6SZc6HYwRB6Q\nzeTVj5mtlSrM7JJLpK6XytKOiYuIKL5sJq92zGzBBGNMeohZ18z+80RZlLiJiCgLsnazibmpN3yf\nVHgKeme4XEQawmXx6nqpjIiI3JHtG0q2AGgVkRCAjQC22cp2AOgCsCuBul4qIyKiLBNu75AZTU1N\n2t7e7nYYRES+IiIdqhpv8i0AbolCREQ+xORFRES+w+RFRES+wzGvDBGRHgCnUniLGgCX0hSOm3Ll\nOgBei1flyrXkynUAqV3LClWtna0Sk5dHiUh7IoOWXpcr1wHwWrwqV64lV64DyM61sNuQiIh8h8mL\niIh8h8nLu9rcDiBNcuU6AF6LV+XKteTKdQBZuBaOeRERke9wv3HKWyKyD8A2s35l+FgQ1p5sIQAN\nANrC5cmWuXgtj5l4DpqYGlR1v9evhSgRTF4e4rcPDb9+OIpIizlvc5TiPQB2qGp4MeY9sNa2TKUs\nY2a5lmoAO83z/apqj8dz1wLc8LMBmHVEU/3jwY2ft1muw1e/NyLSAKAF1jZQGwDsVNVQKvGm5VpU\nlQ+PPADsg/WDDFjbsOxzO6ZZ4t0JazNYjYw13rV45ToB9MHa3ga2WLoi6nSlUubWtZhjrSa2yOOe\nvRbzwRh+3gKgI9WfKTd+3ma5Dl/93nj1/4QTNjzC/CXSqOYvGrX+Cml0N6pZdQGoBFCptr/q412L\nx6+zCdZfgnb9ItKcQpmrVLVfZ/5F68lrMX/hd4W/Vqs10mjbLsnxz5QbP2/xrsMc8tvvzWO22EOJ\nxJSNa2Hy8g7PfgDG46cPxwQEMXOH7F5zPNkyNwVFpMU8dob3pIN3r6UBwO4ox4Pw1x8W8a4DgO9+\nb1aFEw2sawvvZ+jq/wnHvLzDqx+A8QTNmAtg9es/aX4hZ7sWr15nlcPjqZRlw369PjbRCeAQrDGL\ndF9nWqi1c/mG8NciEv5rPDwukuzPVFZ/3ma5DsBnvze2xAUA22HtvQikFm/K18Lk5R3JfKC4zVcf\njgmI9QvUa/5NtswV9g8dtSZfhLuuUrnOjFJV+y7l22FNHOkXEV/9YRHrOszXvvu9MT832wF0qurB\nWc6dlf8Tdht6x2wfKJ4T+eGI6/368a7Fy9fZj5m/QFXmeLJlrjCJqivisP2ve09fS7i1oqrhndWT\n/Zly9ectynX48vdGVUOqugPAYRE5YDu3a/8nbHl5hyc+NBJlfin3qeoq2+FEPhwxS5mb2jHzlypo\nuoGCyZRlKtAE7Yj4Ogir6yoE71/LdlXdbvs6lZ8pN3/ebrgOP/7eiEgw3GpU1f0iss+MT7n6f8Lk\n5R0xPzjdCCZBfv5wnMF0T4XCv6wm1s5Uytyiqp32AXDzfL9ev8/Gs9ciIjthfrbM+avgwz8sYlwH\n4KPfGzM2twfW7Mgbzg3rPjXX/k+YvDzCCx8aTvj8w7EZwGZYv0B7ROSAqobXYtsCoFVEQjA3l9pe\nmmxZxsxyLfvNDbEAUK033qTsuWsBABFpBbDXPA/ffP2M3/6wiHMdIZ/93nTClmxNy7EfwEG3/0+4\ntqGHyI13ndtnIXmSXL/zHrA+HO0/5DGvxW/XSdkhEfdHhamqmPKkfqay/fOWwHX46vfGJNhw0tps\nztuZSrzpuBYmLyIi8h3ONiQiIt9h8iIiIt9h8iIiIt9h8iIiIt9h8iLKASLSLCK7bWvmZeu8ream\n1YbZaxOlD5MXUW4IAthtVkBoFpE+sVaSf0xEOkSkyzzfaZ63iLXVyG4RUfNveMuKx8zru8z9SjGZ\ne8r2whsLK1Me4U3KRLknCGCDbfHXalgb/+0yXz8JYKu5YXYHrPttpheOVdVdIrIRQMh2wzORp7Dl\nRZSDIraxiCzrhzcWQiZKGpMXUe6Jmbgc1iHyLCYvohwTsZdU0nXszDjaATNBoyU8bpZ8lESp4ZgX\nkYeZCRO7w+vimWO7zbF0LszaKiL2rxtwY+ssCGCLWUi1GUC/qu5P4/mJHGHyIvK2g7CtuG0WNG2O\n2OsqHdrsC6OaCRt2IVv5blgLtBK5ht2GRN7WDCuBhT2BmftBZZxtFfGdsG1jT+QWJi8ib9sM4AAw\nvTUF3OquM/eBtYS38OCYF7mJyYvI25rN7rMtsO7Vynqry2YfrI0qwyK3cifKGiYvIo8ySy71msTV\n6fSGYRFpNN18LQCazeoajRHvv9N8OV1mdl9uBtASXmHD/NsPoMHMODyQ6vURpYKbURLlAJPgQmme\ngej5c1P+YsuLiIh8h8mLiIh8h8mLKDf0A9juxpYoAB4y5yfKGo55ERGR77DlRUREvsPkRUREvsPk\nRUREvsPkRUREvsPkRUREvsPkRUREvvP/AZgsyRMH1iCeAAAAAElFTkSuQmCC\n",
```

Comment:
please avoid adding cell outputs, reset notebooks and clear outputs before committing to the repo